jobqueue/client.go

```diff
@@ -211,6 +213,10 @@ func Connect(addr, caFile, certDomain string, token []byte, timeout time.Duratio
 		args:     []string{addr, caFile, certDomain},
 	}
 
+	l := log15.New()
```

Comment:
I think single-letter variable names have their place, but lower case "L" can be hard to distinguish - looks a lot like 1 and i